src/Service.Tests/GraphQLBuilder/MutationBuilderTests.cs

```diff
@@ -1112,7 +1072,7 @@ string[] expectedNames
             // The permissions are setup for create, update and delete operations.
             // So create, update and delete mutations should get generated.
             // A Check to validate that the count of mutations generated is 3.
-            Assert.AreEqual(3 * entityNames.Length, mutation.Fields.Count);
+            Assert.AreEqual(4 * entityNames.Length, mutation.Fields.Count);
```

Comment:
opportunity to clarify comment explaining what `4` signifies. is the existing comment: "/ A Check to validate that the count of mutations generated is 3" still accurate since you changed the number to 4?